src/Service/Resolvers/QueryExecutor.cs

```diff
@@ -191,11 +191,17 @@ public class QueryExecutor<TConnection> : IQueryExecutor
         }
 
         /// <inheritdoc />
-        public virtual string GetSessionParamsQuery(HttpContext? httpContext, IDictionary<string, object?> parameters)
+        public virtual string GetSessionParamsQuery(HttpContext? httpContext, IDictionary<string, DbConnectionParam> parameters)
         {
             return string.Empty;
         }
 
+        /// <inheritdoc/>
+        public virtual void PopulateDbTypeForParameter(KeyValuePair<string, DbConnectionParam> parameterEntry, DbParameter parameter)
+        {
+            return;
```

Comment:
nit: Mention why this is a no-op in the base class

Reply:
Already mentioned in the method summary in the interface. Adding a comment inside the base implementation here.